packages/edit-site/src/components/header-edit-mode/index.js

```diff
@@ -71,12 +72,15 @@ export default function HeaderEditMode() {
 				select( editSiteStore )
 			).getEditorCanvasContainerView(),
 			isDistractionFree: getPreference( 'core', 'distractionFree' ),
+			isFixedToolbar: getPreference( 'core', 'fixedToolbar' ),
 		};
 	}, [] );
 
 	const isLargeViewport = useViewportMatch( 'medium' );
-	const { showFixedToolbar } = useShowBlockTools();
```

Comment:
`showFixedToolbar` was removed from `useShowBlockTools`, so we need to know when to show the fixed toolbar a different way. I'll address the performance of this in a follow-up. Playing around with it in https://github.com/WordPress/gutenberg/pull/60790